performance_test.R

```diff
@@ -0,0 +1,139 @@
+# Challange 2, Autocomplete me
+# Lynn Kaack
+# Performance of the code
+
+library(microbenchmark)
+
+#source("autocomplete.R") # needs to be sourced manually
```

Comment:
Why is that? Without loading `autcomplete.R`, it's not possible to run this file. Requiring users to change source code in order to run a script is less than ideal.